src/portable/portable.c

```diff
@@ -1026,7 +1026,7 @@ int portable_attach(
         return 0;
 }
 
-static bool marker_matches_image(const char *marker, const char *name_or_path) {
+static bool marker_matches_image(const char *marker, const char *name_or_path, bool unversioned_match) {
```

Comment:
hmm, is there a good reason to do version exact matches here ever? maybe e should just losen this up, and always make the version-ignorant checks here?

Reply:
at least i don#t see a reason why we shoud be so strict here.

Reply:
I'm not sure - existing behaviour, mostly. Maybe we want 'detach foo_1.2.raw' to really remove just that? I'll loosen it for now.